<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawWindow</class>
 <widget class="QDialog" name="WithdrawWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
color: white;
}</string>
  </property>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>175</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Määrä</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QWidget" name="nosto_main">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>81</y>
       <width>571</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="withdrawButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nosto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nosto2LineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Takaisin</string>
     </property>
    </widget>
    <widget class="QLabel" name="amount2Label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Määrä</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>backButton</zorder>
    <zorder>amount2Label</zorder>
    <zorder>infoLabel</zorder>
    <zorder>withdrawButton</zorder>
    <zorder>nosto2LineEdit</zorder>
   </widget>
   <widget class="QWidget" name="nosto_kiitos">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>81</y>
       <width>571</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rahat tulossa, kiitos käynnistä</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="nosto">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Valitse nostettava määrä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>81</y>
       <width>571</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Muu</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Takaisin</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_8</zorder>
    <zorder>label_7</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>infoLabel2</zorder>
    <zorder>pushButton_7</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
